tests/unit/core/reducers/test_addons.js

```diff
@@ -201,21 +229,67 @@ describe(__filename, () => {
     });
     const state = addons(undefined,
       loadAddons(createFetchAddonResult(addon).entities));
-    expect(state.byID[addon.id].installURLs).toMatchObject({
-      [OS_MAC]: 'https://a.m.o/mac.xpi',
-      [OS_WINDOWS]: 'https://a.m.o/windows.xpi',
-      [OS_ALL]: 'https://a.m.o/all.xpi',
+    expect(state.byID[addon.id].platformFiles).toMatchObject({
```

Comment:
These tests were only checking the platform logic so I'm not sure you need to match the entire object. I would suggest picking a field that will cover just the platform mapping logic.